core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/OtherSamples.kt

```diff
@@ -16,7 +16,7 @@ class OtherSamples {
 
     @Test
     fun example() {
-        val df = DataFrame.read("../data/titanic.csv", delimiter = ';').take(5)
+        val df = DataFrame.readCSV("../data/titanic.csv", delimiter = ';').take(5)
```

Comment:
Do we need to replace onlhy this call? Did we have something in the docs folder?

Reply:
in theory "no" because all our docs are generated from runnable code, but I'll check anyway :)

Reply:
ha, there was one! Thanks :D